### ISSUE TITLE
GT++ Things I think needs to change

### ISSUE THREAD

**Issue by Bryfer** [GTNewHorizons/NewHorizons#2080]
#### Which modpack version are you using?
2.0.1.0
#
####  Why does this issue exists?
I'd like to start an proper conversation about these items/buildings/things im mentioning here since I personally dont think they fit good into the current stage of the modpack.

If you have opinions about any of these, or anything else GT++ related (_yes we keep this issue GT++ related_)
Feel free to join the discussion.

#
#### GT++ Battery buffers
The Battery buffers dosnt follow the logic from the GT battery buffers, meaning they dont require batteries inside. I also think they provide WAY too much Capacity for their tiers. (Compared to GT)
![image](https://user-images.githubusercontent.com/22825059/31450819-bc317046-aeaa-11e7-8606-c1bad7b2e591.png)
![image](https://user-images.githubusercontent.com/22825059/31450745-88c3d4e2-aeaa-11e7-9ebc-754289c2de75.png)
My suggestion: Disable or reduce capacity. also they might need a recipe upgrade aswell.
#
#### Quad Cell * Battery
Basic concept of them: combine 4 HV batteries to 1 EV battery
Not realistic. you either increase the voltage by connecting batteries one way, or increase capacity by connecting them the other way.
You cant do both at the same time..
There is already EV batteries available from Gregtech and IC2
My Suggestion: Disable.
#
#### Gregtech Computer Cube
Really? is 2 billion eu storage really nessesary inside ONE battery.. at EV tier?.. and for that recipe?
![image](https://user-images.githubusercontent.com/22825059/31449549-7c2591ec-aea7-11e7-8dd9-0ca1ad4363e0.png)
My Suggestion: Disable.

#
#### GT++ Bee frames
Is this even needed? Isnt bees powerful enough already once you get them going?
I havent tried them yet. but based on their text its apperently more than im expecting?
![image](https://user-images.githubusercontent.com/22825059/31449292-cdc699ac-aea6-11e7-9a79-2f4acdcbbd8c.png)
My Suggestion: Disable.
#
#### GT++ Tunnel Digger, Tree Feller (and Cobblestone Breaker)
These tools are not needed since we already have TiCo tools available. and should be using TiCo tools.
![image](https://user-images.githubusercontent.com/22825059/31449723-f75a580c-aea7-11e7-88a2-6828e53ad4c0.png)
My Suggestion: Disable.
#
#### GT++ Tree Farmer
Not everyone has tried the Tree Farmer yet.
Right now we have more than 4? Tree Farmers available from different mods.. yey?
This farm looks like this when working (see picture below)
Worth mentioning about it. You need a GT Saw inside the controller block.
You also need Maintenance Hatch, Output Bus, Energy Hatch, Controller, Input Hatch, Input Bus
You also need to feed the Tree Farmer with Fertiliser Cell (item id 9103) for it to work properly.
![image](https://user-images.githubusercontent.com/22825059/31451198-cc5961ee-aeab-11e7-8837-a14dbfb925d8.png)
![image](https://cdn.discordapp.com/attachments/234936569360809996/367519081257238548/unknown.png)
![image](https://cdn.discordapp.com/attachments/255143237218467840/367522405339627520/unknown.png)
Yes, the tooltip needs to be updated. also why is the capacity so large??
My Suggestion: reduce capacity, update tooltip and see if we need this 4:th tree farm ingame.
#
#### GT++ Advanced Boiler
_I havent had time to test this properly myself, go here instead_
https://github.com/GTNewHorizons/NewHorizons/issues/1929
#
#### GT++ Fish Catcher
_I havent had time to test this properly myself, go here instead_
https://github.com/GTNewHorizons/NewHorizons/issues/2051
#

**Comment by AlphaApotheosis** [comment]
We should disable the things that are entirely duplicative with multiple other options. So remove the tree farmer, quad cell batteries, and GT++ battery buffers. There are different batteries available at EV+, and the gregtech way is to put them in battery buffers.

The computer cube and the bee frame seem silly, and should be at least seriously nerfed, if not disabled.

**Advanced Boilers:** Based on my earlier testing, these produce about 10x or more steam per fuel than the GT boilers. Output has been reduced in 2.0.1.0 to 33%, but efficiency is still higher than GT boilers. In addition, and I think this is a bigger problem - they can use all furnace fuels, unlike GT boilers. This includes BoP Ash from the nether, diamonds, and many other things. This makes it very difficult to balance and I suggest these be disabled as well.

**GT++ Multiblock Sifter:** These have an increased output chance for many versions of the "rare" gems. It seems that  for some exquisite gems that would otherwise have only a 3% chance, the chance is 10% or 20%. This seems highly unbalanced, and doesn't make a lot of sense. 

It makes sense for a multiblock sifter to be substantially faster, maybe more energy efficient than the single block sifters, but I don't think large increases in rare output makes any kind of sense at all. They should be disabled or the output chances set to match the single block sifters).

**Comment by KorGgenT** [comment]
I disagree that the tree farmer should be disabled only because there are other options available. Why pigeonhole people into a single way of doing things? 

I will be testing the Tree Farmer later today. I will comment again or edit this comment with more information.

**Comment by Dream-Master** [comment]
I dont like disable things except the Tools. I can change recipes capacities or different things.

**Comment by Dream-Master** [comment]
Have to check that tree farm from gt++ comes after Forestry tree farm

**Comment by AlphaApotheosis** [comment]
@Dream-Master I understand you don't like to disable things, but I think a problem is that there are no config options or recipe solutions to some of these issues. Does it make sense that the GT++ advanced boilers use furnace fuels (BoP Ash? Diamonds?) when no GT boilers can? Can you balance a boiler that is not using GT fuel values with GT boilers?

Also the GT++ multiblock sifter production of better gems, does that make sense? I don't think so.

I think some of these items just don't fit in the modpack at all. They are supposedly gregtech related, but function in a very non-greggy way. I think that people should have to use batteries to store energy in battery buffers, instead of building a block that does not need them.

**Comment by KorGgenT** [comment]
All right, I've had a chance to examine the Battery Buffers and the Energy Buffers. I completely disagree with Alpha. I think they are balanced. Here's why:


1)
The very minimum tier you need to be able to make them is the same tier as MFEs. So you need to be able to make stainless steel, energetic alloy, energy crystals. 

This also makes the energy buffers *very expensive*. It costs 2^t MFEs to make an energy buffer, where t = the tier, starting with ULV at 0.


2a)
Battery Buffers are Versatile. You can have a battery buffer output anywhere from 1A to 16A, and you can take batteries out and shuffle them around. You lose very little energy of note when you do this.

You can even put batteries right into machines!

2b)
Energy Buffers are not Versatile. Where you place one is where it will stay. If you break it, you lose the energy inside.
It is always 4A. 


3)
From my experience, what makes a thing "greggy" is that it's a machine that's rather specialized. You have assemblers, disassemblers, ;athes, extruders, fluid solidifiers and extractors, canners, etc.
From this, you can see that a battery buffer and an energy buffer serve somewhat of a different purpose: For a (much) larger investment in resources, you can make a permanent installation that will store a lot of energy. This is not unlike a railcraft tank for fluids.
For a relatively small investment in resources, you can make a nonpermanent installation that will store some energy. This is not unlike Iron tanks with the dolly, etc.




From all this, I urge you not to change the energy buffer.

**Comment by SirAzazel** [comment]
@Dream-Master leaving those quad batteries in? are you serious? what are they? pseudo-paralel or something? as noted by bryfer, batteries can be connected in series to boost voltage or in parallel to boost capacity. CANNOT DO BOTH AT SAME TIME, PHYSICALLY NOT POSSBILE ![](http://www.12voltbattery.info/images/content/batteryWireDiagram.jpg)

**Comment by SirAzazel** [comment]
My oppinion (sorted same as OP)
--
Battery buffers: They ARENT buffers as they are, they are supercapacitors or something. Nerf storage so they have 10 times their output and then they are buffers, otherwise they are unbalanced pieces of crap
# 
Quad cells: Read above. Unbalanced counter-physical thing made by person that most likely doesnt understand anything in electricity.
# 
Digger and Feller: same as with bryfer, they arent needed as we already have TiCo alternatives which are better and whole pack makes you use TiCo anyway, why those must be excluded?
# 
Farm: glitchy garbage that didnt work properly before since probs was never even tested enough before releasing (not even noting that it has far superior alteranatives)

Doesnt cut whole tree if its not in bounds
![](https://i.imgur.com/00D1Z1C.jpg)

Saw goes below 0 durability (was patched i think)
![](https://i.imgur.com/f93RmSy.png)
# 
Adv Boiler: idk what its now after fix of steam rates
# 
Fish catcher: same here

**Comment by Technus** [comment]
Quad cells: as noted by bryfer, batteries can be connected in series to boost voltage or in parallel to boost capacity. CANNOT DO BOTH AT SAME TIME, PHYSICALLY NOT POSSBILE . 
Possible, if there is some switching CIRCUIT and inductance

**Comment by Technus** [comment]
Also here the EU is energy capacity not rated voltage.

**Comment by Technus** [comment]
Starting from this equation that compares E - Energy capacity (EU), V - rated Voltage (EU/t), i can get Q - charge and C - capacity
![image](https://user-images.githubusercontent.com/4737429/31530882-be4d90b4-afe3-11e7-98f4-bc4422bc0ecf.png)

we have C=(2*E)/(V^2)
and E=(C*(V^2))/2

ex.
1000EU 32EU/t 1A -> True Capacity value = 2*1000[EU]/32[EU/t]^2 = 1,953125

in parallel we add capacity
1,953125 + 1,953125 +1,953125 +1,953125  = 1,953125 * 4 = 7.1825
Computing energy back we have
(7.1825*32[EU/t]^2)/2= 4000 EU, WOW HOW EXPECTED!

in series we have this formula
![image](https://user-images.githubusercontent.com/4737429/31530608-01bf1d6a-afe2-11e7-92cf-eef878543350.png)

but for same capacity we can just divide by the count of batteries in series , also rated voltage goes 4x
1,953125 / 4 = 0,48828125
lets compute energy it can store
(0,48828125*128[EU/t]^2)/2= 4000 EU, WOW HOW UNEXPECTED!



In short EU does not refer to Capacity but to Energy Storage!

**Comment by Technus** [comment]
Another ex. Lets say charge every batt to full

How come making that in series destroys energy?

**Comment by KorGgenT** [comment]
What you should be asking is this: "Is this item balanced in the game?" 
Not: "Is this item perfectly realistic?"

**Comment by unexpectedbukakke** [comment]
I think all the GT stuff listed here is fine. Maybe the tree farmer should be disabled due to bugginess, I don't know. The energy buffers are really niche since as already mentioned they are limited in terms of their usefulness with respect to portability and amperage, and also in my opinion they have more difficult recipes than regular battery buffers.

I have not seen anyone talk about this yet, and I have not built one of these myself, but a cursory look at the GT++ Alloy Blast Smelter makes it seem very overpowered. It makes liquid metals which might be why it is easily overlooked, but check out how to make Draconium Ingots in the GT++ blast smelter versus the EBF.

GT++ Alloy Blast Smelter: 120 EU/t required, total 1.38 million EU consumed
GT EBF: 30,720 EU/t required, total 122 million EU consumed, requires Argon gas, requires Naquadah coils, produces hot ingots which must be cooled down in a vacuum furnace,

**Comment by Dream-Master** [comment]
So far i know did not working all materials over 3600k in the multi smelter.

**Comment by Bryfer** [comment]
So.. based on the comments I've read sofar, it seems like I'm (almost) the only one who think the internal battery capacity of GT++ machines is ridiculously high.

Have fun feeding high tiered voltage for 3.5hours into each GT++ machine you build in the future.

I believe the internal capacity of normal GT machines is good, please compare them to GT++ by yourselves. 
I'm personally dropping out from this discussion since it seems I'm the only one thinking of properly balancing machines.

**Comment by Dream-Master** [comment]
I will look at it when I am back and see what I can change

**Comment by Methes** [comment]
W-wait. 3.5 hours of lets say 2048EU/t is 516M EU. Does really every GT++ machine have its buffer this high? Not just the battery buffers? That's a serious trouble then.

**Comment by DeForce** [comment]
@Bryfer looking at the number of computer cube just looks someone accidentally max-inted somehow.
